File: Algorithms/Implementation/Grading Students/gradingStudents.cs
//Problem URL: https://www.hackerrank.com/challenges/grading/problem

public static List<int> gradingStudents(List<int> grades)
{
	for(int i = 0; i < grades.Count(); i++){
		if(grades[i] >= 38){
			int difference = 5 - (grades[i] % 5);
			if(difference < 3){
				grades[i] += difference;
			}
		}
	}
	return grades;
}

File: Algorithms/Implementation/Grading Students/gradingStudents.py
#Problem URL: https://www.hackerrank.com/challenges/grading/problem

def gradingStudents(grades):
    for x in range (0, len(grades)):
        if(grades[x] >= 38):
            difference = 5 - (grades[x] % 5)
            if(difference < 3):
                grades[x] += difference
    
    return grades
